src/buildHTML.js

```diff
@@ -654,10 +654,60 @@ export default function buildHTML(tree, options) {
     // normal place) so we use an absolute value for vertical-align instead
     bottomStrut.style.verticalAlign = -body.depth + "em";
```

Comment:
I didn't realize that you could pass `verticalAlign` a length.  It makes me wonder why the `topStrut` is even necessary since the `bottomStrut`'s height is already `body.height + body.depth`.

Reply:
Hmm, good question! I should experiment with removing the topStrut.